### DISCUSSION TITLE
I have a big problem with this platform HELP!!!

### DISCUSSION THREAD

**Discussion by Booker141** [vercel/community#3073] [Help]
### Summary

I'm developing a next.js social media app and when I post a new post when it redirects me to the new page it doesn't load new post until I reload from to few times. I don't know what to do. I'm using MongoDB. This problem is happening with create, edit and delete methods of all my entities. Any help pls! I need to finish this project by this Monday. Thanks in advance.

### Example

https://sweet-home-bay.vercel.app/

### Steps to Reproduce

You can try to post in the app link

**Comment by amyegan** [comment]
You may be able to find more details in the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#) as well as your browser's dev tools console and network tab.

If you share your public repo or minimal reproduction, someone here may be able to recognize what is going wrong. 🙏

**Reply by Booker141** [reply]
Hi! Thank you so much for asking. It's a private project but I have the build link which is: https://sweet-home-bay.vercel.app/
